pages/item-new/_.vue

```diff
@@ -0,0 +1,369 @@
+<template>
+  <div>
```

Comment:
Would it make sense to add a css class here for the purpose of keeping all the new styling together & potentially away from interfering with the current layout?

Reply:
I am not sure if that is necessary. If we need specific CSS, it can be scoped here without breaking the old item page. Did you have a specific use case in mind?

Reply:
The rounded corners on the metadata box was one example, but the way you've done it by adding the styling to this page only will work fine too.